requirements/base.in

```diff
@@ -1,7 +1,9 @@
 # Core requirements for using this package
 -c constraints.txt
 
+django-pyfs>=1.0.5
```

Comment:
Anything that depends on Django can't be in the core dependencies; the intention is for the default XBlock installation to not depend on Django, with dependencies for the optional Django support only being installed via `pip install XBlock[django]`.

Reply:
Removed from base.in, I see they are already present in installation requirements in `setup.py`